fli-iam/shanoir-ng
# [fli-iam#1864] Add NIRS type BIDS

NIRS BIDS files can now be imported into Shanoir

Specs : https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/11-near-infrared-spectroscopy.html

How to test :
- Import NIRS modality BIDS into Shanoir through the BIDS import
- Check the created examinations/acquisitions/datasets

Data samples : 
https://osf.io/juxy8
https://github.com/bids-standard/bids-examples#nirs-datasets
